src/pages/BlogCollectionAdminPage.tsx

```diff
@@ -90,6 +107,14 @@ const BlogAdminCollectionPage: React.FC = () => {
               .catch(handleError)
           }
         />
+        <OrderExportModal
+          renderTrigger={({ setVisible }) => (
+            <StyledButton onClick={() => setVisible(true)}>
+              <ExportIcon className="pr-1" />
+              {formatMessage(blogMessages.text.exportButton)}
+            </StyledButton>
+          )}
+        ></OrderExportModal>
```

Comment:
Becuase use React.VFC, you don't need close tag.
You can rewirte component like this: <OrderExportModal ...props />

Reply:
change it